xmodule/modulestore/xml_exporter.py

```diff
@@ -282,6 +286,15 @@ def process_extra(self, root, courselike, root_courselike_dir, xml_centric_cours
 
         _export_drafts(self.modulestore, self.courselike_key, export_fs, xml_centric_courselike_key)
 
+        courselike_key_str = str(self.courselike_key)
+        block_id_pattern = f"{courselike_key_str.replace('course-v1:', 'block-v1:', 1)}*"
+
+        tags_count = get_object_tag_counts(block_id_pattern)
+        course_tags_count = get_object_tag_counts(courselike_key_str)
+
+        if tags_count or course_tags_count:
```

Comment:
I found the cause of having an empty `tags.csv` in my tests. At the auto-tagging handler, we tag the course (`course-v1:...`) and the "course block". The latter is not shown anywhere but probably appears in this count.

Could we create a small task to fix this @bradenmacdonald?

Reply:
@rpenido Sure, thanks!